docs/manuals/source/Appendix/BackwardCompatibility.rst

```diff
@@ -29,26 +29,18 @@ Compatibility between Bareos and Bacula
 :index:`\ <single: Compatibility; Bacula>`
 :index:`\ <single: Bacula>`
 
-A Director and a Storage Daemon should (must) always run at the same version. This is true for Bareos as well as for Bacula. It is not possible to mix these components. This is because the protocol between Director and Storage Daemon itself is not versioned (also true for Bareos and Bacula). If you want to be able to switch back from Bareos to Bacula after using a Bareos director and storage daemon you have to enable the compatible mode in the Bareos storage daemon to have it write the data in
-the same format as the Bacula storage daemon.
-
-The |fd| is compatible with all version of the Bacula director (tested with version 5.2.13 and lower) when you enable the compatible mode in the configuration of the |fd|. The compatible option was set by default in Bareos < 15.2.0, and is disabled by default since :sinceVersion:`15.2.0: Compatible = no`.
-
-To be sure this is enabled you can explicitly set the compatible option:
-
-:config:option:`fd/client/Compatible = True`\
+A Director and a Storage Daemon should (must) always run at the same version. This is true for Bareos as well as for Bacula. It is not possible to mix these components. This is because the protocol between Director and Storage Daemon itself is not versioned (also true for Bareos and Bacula).
```

Comment:
I would say we can get rid of the whole chapter of bacula compatibility. It is not needed anymore.